### DISCUSSION TITLE
Infinix_INBook_Hackintosh- Discussion!

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with this repository and to me:
  * Ask questions you’re wondering about.
  * Share ideas related to this repo.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build 💪.
#  If any issue, report on issues section.
[Switch to issue section](https://github.com/devboloji/Infinix-Hackintosh-Guide-Opencore_INBook_X1_XL11_i3-/issues)

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by babibudo** [comment]
Hi, first of all thank you very much for the efi. Helps a lot during the instalation of big sur on my infinix x1. I want to ask about the hdmi port output, does it work?

**Reply by devboloji** [reply]
In hackintosh, most of the devices HDMI and VGA won't work 99%. Because apple natively doesn't supported VGA. Think like you have mac, so buy a dock which is type c . But remember in every hackintosh device hdmi won't work 99%. Here, by this efi hdmi won't work.. but dock works
In future, hdmi may be fixed not 100%

**Reply by babibudo** [reply]
I see, than you very much for your reply, really appreciated it.

**Reply by babibudo** [reply]
Hi, sorry for asking about this again. I have bought a type-c to hdmi adapter (https://www.ventioncable.com/product/type-c-to-hdmi-adapter/) and try to display out to external monitor but I can't get it t work. It works fine on windows though, is it possible to make it works on macos using this EFI?

**Reply by devboloji** [reply]
what macOS version are you using?

**Reply by babibudo** [reply]
Currently using big sur 11.6.7, anyway thank you for the updated EFI...have been using it and work great.

**Reply by devboloji** [reply]
go to `system Preferences` -> `Display` -> hold `command or alt` and see you see `Detect Displays` then enter it...
If you tried it, Its ok.....or If it work remind here so its helpful to anyone..
anyway, You are welcome...
Have to tried the latest efi >?? Found the issues...?

**Reply by babibudo** [reply]
Nothing happened when I click on the "Detect Displays", still no external output display (using the latest EFI right now). I guess I just have to use windows if I want to use external display. Is the external output display working fine in your setup? The latest EFI is great, imessage and facetime works fine too, other than display output in my case there's no issue so far.

**Reply by devboloji** [reply]
I don't use external monitor for now, but I will buy the external monitor adapter later to test in Hackintosh. But I will sure test. It Takes time.. Try with a dock which supports windows and Mac.

**Reply by babibudo** [reply]
Sure, thank you very much for the assistance.

**Comment by babibudo** [comment, marked as answer]
Hi, I want to ask that if you play a video on apple TV+, free episode for example. Then macos will freeze when the video start, and the only way to get out from it is by holding the power button until the laptop restarted. If I only play trailer on the apple tv app, it works fine though. Does it occured to you too?

**Reply by babibudo** [reply]
I think it's related to the DRM issue, cannot play netflix on safari too. As a workaround to watch netflix and apple tv is by using firefox.

**Reply by devboloji** [reply]
Yes.. Tried with Chrome?

**Reply by babibudo** [reply]
Yup, chrome works fine too.

**Reply by devboloji** [reply]
The problem is not only for Infinix. This issue is common in many other devices. Tested in my infinix laptop & Lenovo device. So, no need to worry.

**Reply by devboloji** [reply]
@babibudo Can You play spider man far from home trailer?in safari?
